scripts/tools/EXT_Exp_MetricProgrammable.rst

```diff
@@ -0,0 +1,123 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+    t=tags['$t']
+    T=t.upper()
+%>
+:orphan:
+
+.. _ZET_experimental_programmable_metric:
+
+==========================================
+Metric Programmable Experimental Extension
+==========================================
+
+API
+----
+* Enumerations
```

Comment:
where are new handle types defined?

Reply:
Those are defined in metricProgrammable.yml
https://github.com/oneapi-src/level-zero-spec/pull/204/files#diff-5f98b851645e6662ea9e2b7ac56a0566cd73f10f5bf2fe42b91db481bb4723a0